.github/workflows/release-deploy-all.yml

```diff
@@ -33,6 +33,37 @@ on:
       deployFormioDefinitions:
         required: true
         type: string
+    secrets:
+      SA_TOKEN:
```

Comment:
This ensures that the required secrets and in place before the workflow starts right? I meant if any of these secrets are missing then it will fail upfront?

Reply:
That is the idea. It also does not work if they are not present. I will be able to test it once it is merged into the main.